### PR TITLE
ssh key verify code update

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
Signed-off-by: Prajyot Parab <prajot.parab@ibm.com>

### CODE REVIEW

File: openshift-install-powervs [hunk 1]
```diff
@@ -877,19 +882,17 @@ function variables {
     debug_switch
   fi
 
-  if [ -f "./id_rsa" ] && [ -f "./id_rsa.pub" ]; then
-    log "Found id_rsa & id_rsa.pub in current directory"
-    cp -f ./id_rsa ./id_rsa.pub ./"$ARTIFACTS_DIR"/data/
-  elif [ -f "$HOME/.ssh/id_rsa" ] && [ -f "$HOME/.ssh/id_rsa.pub" ]; then
+  if [ -s "./id_rsa" ] && [ -s "./id_rsa.pub" ]; then
+    echo "private_key_file = \"$PWD/id_rsa\"" >> $VAR_TEMPLATE
```
Comment (yussufsh) [inline openshift-install-powervs:886]: Can you please keep this log statement.

File: openshift-install-powervs [hunk 2]
```diff
@@ -481,15 +481,14 @@ function verify_data {
...
-  elif [ -f "./id_rsa" ] && [ -f "./id_rsa.pub" ]; then
-    log "Found id_rsa & id_rsa.pub in current directory"
-    cp -f ./id_rsa ./id_rsa.pub ./"$ARTIFACTS_DIR"/data/
-  else
-    warn "Creating new SSH key-pair..."
-    ssh-keygen -t rsa -f ./id_rsa -N ''
-    cp -f "./id_rsa" "./id_rsa.pub" ./"$ARTIFACTS_DIR"/data/
+  if [ ! -s "$VAR_PRIVATE_KEY_FILE" ] && [ ! -s "$VAR_PUBLIC_KEY_FILE" ]; then
+    if [ -s "./id_rsa" ] && [ -s "./id_rsa.pub" ]; then
+      cp -f ./id_rsa ./id_rsa.pub ./"$ARTIFACTS_DIR"/data/
```
Comment (yussufsh) [inline openshift-install-powervs:486]: Can you keep the log statement here.. this will be when user provides all variables except the key files right?

Comment (Prajyot-Parab) [reply openshift-install-powervs:486]: yes

**Review by yussufsh — APPROVED** [review]
Thanks @Prajyot-Parab

**Pull request merged**